### PR TITLE
Event Section for Users

### PR DESCRIPTION
**Pull request by anirudhp26**
## Changes proposed
Event Button added in Navbar

## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/1820"></a>

### CODE REVIEW

**Comment by anirudhp26** [comment]
Please don't close this PR as all the changes done for the Feature will be committed here

File: data/anirudhp26.json
Comment (krshkodes) [inline data/anirudhp26.json:44]:
```suggestion

```

**Review by krshkodes — COMMENTED** [review]
I don't get how this adds any value to someone's profile? 🤔

**Comment by krshkodes** [comment]
Just saw the issue comments, and seems justified, my bad

**Comment by anirudhp26** [comment]
Hey @eddiejaoude , Sorry for the mention 
The Event feature you proposed is complete 

## Procedure to add an Event
: Users need to edit their data file and add an event section like I have in my file ( anirudhp26.json ), However the fields mentioned must be filled for event to get mentioned on the event page

image

## Users can mention more than 1 events

**Comment by anirudhp26** [comment]
@krshkodes Please do let me know if the design of the event page is good !
Feel free to change it if needed

**Comment by anirudhp26** [comment]
## Here's the current design

image

**Comment by anirudhp26** [comment]
@eddiejaoude Any updates sir?

**Comment by eddiejaoude** [comment]
Sorry @anirudhp26 I missed the previous notifications, so thank you for the nudge 👍  I will take a look now

File: components/events/EventPreview.js
Comment (eddiejaoude) [inline components/events/EventPreview.js:14]: this can be achieved with TailWindCSS, I don't think custom styles are required

File: pages/events.js
Comment (eddiejaoude) [inline pages/events.js:38]: what does `eve` meant?

File: pages/events.js
Comment (eddiejaoude) [inline pages/events.js:33]: I think you could use

```js
if (user.events) {
```

File: components/events/EventPreview.js
Comment (eddiejaoude) [inline components/events/EventPreview.js:31]: I think TailWind can do all of this

Comment (anirudhp26) [reply components/events/EventPreview.js:31]: Okay I'll try resolving this suggesttions

**Comment by anirudhp26** [comment]
@eddiejaoude , I replaced the styles with Tailwind-CSS, and also made the events mapping more clear and understandable,  please have a look

**Comment by eddiejaoude** [comment]
Thank you! 👍  Could you update your branch with the `nextjs` branch so I can test it on Gitpod

**Comment by anirudhp26** [comment]
Which branch? 
I created pull request in LinkFree's nextjs branch ...!

**Comment by eddiejaoude** [comment]
What you have done is correct 👍 , but there were some other changes that went into `EddieHubCommunity:nextjs` branch, could you update your forked main branch and then merge it into your forked branch `anirudhp26:nextjs`

I made a video on it but any questions let us know https://www.youtube.com/watch?v=FnxFwyzm4Z4

**Comment by anirudhp26** [comment]
Done sir 👍

File: pages/events.js
Comment (eddiejaoude) [inline pages/events.js:32]: Note for a future issue/pr, the list should be in date order, not user order. My suggestion would be to do this on the server side and here just display the list of events.

This is not blocking the PR though.

**Review by eddiejaoude — APPROVED** [review]
Thanks 👍

File: pages/events.js
Comment (anirudhp26) [reply pages/events.js:32]: Sure sir, I'll look into it

Comment (eddiejaoude) [reply pages/events.js:32]: If you get time that would be great 👍 , but it can come in another issue and PR. I will merge this soon

**Comment by anirudhp26** [comment]
Okay so should I raise another issue for that? And create another PR?

**Comment by eddiejaoude** [comment]
You could do it in this issue and PR if you prefer, but also another one is fine also 🤓

**Comment by anirudhp26** [comment]
Hey @eddiejaoude  sorry for the delay, The events display in order feature would take time to figure out ig 

So if you can merge this PR, and once I figure out how to update the events according to the Dates, it would be great ✨

**Comment by anirudhp26** [comment]
That's a great news sir ✨ 
Thanks for guiding me through my first PR on a big project like this

**Comment by eddiejaoude** [comment]
Thank you for pushing this feature forwards 💪

**Pull request merged**
